docs/flow/concepts/configuration_language.md

```diff
@@ -61,6 +61,17 @@ supports more complex expressions, such as:
 
 Expressions may be used for any attribute inside a component definition.
 
+### Referencing component exports
+
+The most common expression is to reference the exports of a component like
+`local.file.password_file.content`. A reference to a component's exports is
+formed by merging the component's name (e.g., `local.file`), label (e.g.,
+`password_file`), and export name (e.g., `content`), delimited by period.
+
+For components that don't use labels, like
+`prometheus.integration.node_exporter`, only combine the component name with
+export name: `prometheus.integration.node_exporter.targets`.
```

Comment:
We don't actually have a merged component which doesn't use a label right now, so we might need to remove this by v0.28 if #1872 doesn't get merged.